lib/manifold/cli.rb

```diff
@@ -47,14 +45,12 @@ def add(name)
       logger.info "Added workspace '#{name}' with tables and routines directories."
     end
 
-    desc "generate PROJECT_NAME SERVICE", "Generate services for a project"
-    def generate(project_name, service)
-      case service
-      when "bq"
-        bq_service.generate_dimensions_schema(project_name)
-      else
-        logger.error("Unsupported service: #{service}")
-      end
+    desc "generate", "Generate BigQuery schema for all workspaces in the project"
+    def generate
+      name = Pathname.pwd.basename.to_s
+      project = API::Project.new(name, directory: Pathname.pwd, logger: logger)
```

Comment:
Just calling out this awkwardness explicitly. I think we should assume we're running this command in a project directory, so the name of the project would be the basename of the working directory.

I think there's an argument to be made that projects should just consider a directory in their constructors, and the project name is always the basename of that directory. Started to pull that thread in this PR and decided against it.

Reply:
I see no awkwardness. Well, nothing egregious anyway. Your assumption is entirely reasonable 🙂